stdlib/public/SDK/Foundation/NSError.swift

```diff
@@ -469,14 +469,9 @@ public protocol _BridgedStoredNSError :
   init(_nsError error: NSError)
 }
 
-/// TODO: Better way to do this?
 internal func _stringDictToAnyHashableDict(_ input: [String : Any])
     -> [AnyHashable : Any] {
-  var result = [AnyHashable : Any](minimumCapacity: input.count)
-  for (k, v) in input {
-    result[k] = v
-  }
-  return result
+  return input as [AnyHashable: Any]
```

Comment:
Since this expression is so simple now, we should just scrap this whole helper function and write the above in the 2 places it's called.

Reply:
(if the cast is even necessary, it might do it implicitly)

Reply:
Good point, it should be implicit now.

Reply:
Agreed. Updated PR.